core/src/upickle/core/BufferingInputStreamParser.scala

```diff
@@ -63,13 +63,13 @@ trait BufferingInputStreamParser{
     //
     // - https://github.com/scala-js/scala-js/issues/3913
     val untilBufferOffset = until - firstIdx + 1
-    if (untilBufferOffset >= buffer.size){
+    if (untilBufferOffset >= buffer.length){
```

Comment:
Seems like the jvm should inline this away, but it's still showing up under profiler and easy to fix.

![buffer size](https://user-images.githubusercontent.com/663139/72023689-77a84180-3241-11ea-8d1b-f8fae14e07ce.png)